src/lib/coda_lib/coda_lib.ml

```diff
@@ -766,6 +768,8 @@ module Make (Inputs : Inputs_intf) = struct
                       in
                       (Some persistence, root_snarked_ledger, frontier) )
             in
+            (* TODO: a proper name from Config should be the name of the database *)
```

Comment:
did you intend to fix this? if not, can you open an issue and add the number here?